h2o-py/h2o/automl/autoh2o.py

```diff
@@ -303,6 +327,8 @@ def train(self, x = None, y = None, training_frame = None, fold_column = None,
         :param blending_frame: H2OFrame used to train the the metalearning algorithm in Stacked Ensembles (instead of relying on cross-validated predicted values).
             This is optional, but when provided, it is also recommended to disable cross validation 
             by setting `nfolds=0` and to provide a leaderboard frame for scoring purposes.
+        :param verbosity: Verbosity of the backend messages printed during training.
+            Available options are 'debug', 'info' or 'warn'. Defaults to None (disable live log).
```

Comment:
@ledell I've been hesitating to default verbosity param to 'warn' level, same for R API. Any preference?